integration/setup/bastion_instance.go

```diff
@@ -0,0 +1,565 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/microerror"
+)
+
+// TODO make the resource management more reliable to ensure proper setip and
```

Comment:
typo: setup

Reply:
You can suggest an edit which makes it super easy for me to apply it. Anyway, sorted. Thanks for noting.